Formula/docker-compose.rb

```diff
@@ -16,137 +16,20 @@ class DockerCompose < Formula
-    sha256 "2d475327684562c3a96cc71adf7dc8c4f0565175cf86b6d7a404ff4c771f15f0"
-  end
-
-  resource "PyNaCl" do
-    url "https://files.pythonhosted.org/packages/cf/5a/25aeb636baeceab15c8e57e66b8aa930c011ec1c035f284170cacb05025e/PyNaCl-1.4.0.tar.gz"
-    sha256 "54e9a2c849c742006516ad56a88f5c74bf2ce92c9f67435187c3c5953b346505"
-  end
-
-  resource "pyrsistent" do
-    url "https://files.pythonhosted.org/packages/4d/70/fd441df751ba8b620e03fd2d2d9ca902103119616f0f6cc42e6405035062/pyrsistent-0.17.3.tar.gz"
-    sha256 "2e636185d9eb976a18a8a8e96efce62f2905fea90041958d8cc2a189756ebf3e"
-  end
-
-  resource "python-dotenv" do
-    url "https://files.pythonhosted.org/packages/66/94/286b7b88e24a7a2f3e020234524e3dc3dfdba313f299054b24b8d4d39b77/python-dotenv-0.17.1.tar.gz"
-    sha256 "b1ae5e9643d5ed987fc57cc2583021e38db531946518130777734f9589b3141f"
-  end
-
-  resource "PyYAML" do
-    url "https://files.pythonhosted.org/packages/a0/a4/d63f2d7597e1a4b55aa3b4d6c5b029991d3b824b5bd331af8d4ab1ed687d/PyYAML-5.4.1.tar.gz"
-    sha256 "607774cbba28732bfa802b54baa7484215f530991055bb562efbed5b2f20a45e"
-  end
-
-  resource "requests" do
-    url "https://files.pythonhosted.org/packages/6b/47/c14abc08432ab22dc18b9892252efaf005ab44066de871e72a38d6af464b/requests-2.25.1.tar.gz"
-    sha256 "27973dd4a904a4f13b263a19c866c13b92a39ed1c964655f025f3f8d3d75b804"
-  end
-
-  resource "six" do
-    url "https://files.pythonhosted.org/packages/71/39/171f1c67cd00715f190ba0b100d606d440a28c93c7714febeca8b79af85e/six-1.16.0.tar.gz"
-    sha256 "1e61c37477a1626458e36f7b1d82aa5c9b094fa4802892072e49de9c60c4c926"
-  end
-
-  resource "texttable" do
-    url "https://files.pythonhosted.org/packages/f5/be/716342325d6d6e05608e3a10e15f192f3723e454a25ce14bc9b9d1332772/texttable-1.6.3.tar.gz"
-    sha256 "ce0faf21aa77d806bbff22b107cc22cce68dc9438f97a2df32c93e9afa4ce436"
-  end
-
-  resource "urllib3" do
-    url "https://files.pythonhosted.org/packages/cb/cf/871177f1fc795c6c10787bc0e1f27bb6cf7b81dbde399fd35860472cecbc/urllib3-1.26.4.tar.gz"
-    sha256 "e7b021f7241115872f92f43c6508082facffbd1c048e3c6e2bb9c2a157e28937"
-  end
-
-  resource "websocket-client" do
-    url "https://files.pythonhosted.org/packages/48/bf/c706b56243f1641159ff211b51d3341024e1cdf63defc2b74595b6319039/websocket-client-0.59.0.tar.gz"
-    sha256 "d376bd60eace9d437ab6d7ee16f4ab4e821c9dae591e1b783c58ebd8aaf80c5c"
-  end
-
   def install
-    virtualenv_install_with_resources
-
-    bash_completion.install "contrib/completion/bash/docker-compose"
-    zsh_completion.install "contrib/completion/zsh/_docker-compose"
+    ldflags = %W[
+      -s -w
+      -X github.com/docker/compose/v2/internal.Version=#{version}
+    ].join(" ")
+    system "go", "build", *std_go_args(ldflags: ldflags), "./cmd"
   end
 
   test do
-    system bin/"docker-compose", "--help"
+    output = shell_output(bin/"docker-compose compose up 2>&1", 14)
+    assert_match "can't find a suitable configuration file", output
```

Comment:
```suggestion
    expected_message = "can't find a suitable configuration file"
    output = shell_output("#{bin}/docker-compose compose up 2>&1", 14)
    assert_match expected_message, output

    output = shell_output("#{Formula["docker"].opt_bin}/docker compose up 2>&1", 14)
    assert_match expected_message, output
```

Reply:
ah this almost works, but `docker` searches specific locations for plugins ([ref](https://github.com/docker/compose#linux)):

1. ~/.docker/cli-plugins/docker-compose
1. /usr/lib{,exec}/docker/cli-plugins
1. /usr/local/lib{,exec}/docker/cli-plugins

so we need to symlink `docker-compose` into one of these, is that possible?

Reply:
Not on ARM or linux.

Reply:
with `2.0.1`, `docker-compose` can now be called directly meaning it's usable without the symlink. however, do we think a caveat is applicable here to specify how to enable `docker compose`? I think so because [Archlinux](https://github.com/archlinux/svntogit-community/blob/7360ecb6027f5c474f6b6e5560624176d14c07cd/trunk/PKGBUILD#L38) is able to install the symlink